staging/src/k8s.io/kube-scheduler/config/v1alpha1/types.go

```diff
@@ -198,9 +200,13 @@ type Plugins struct {
 type PluginSet struct {
 	// Enabled specifies plugins that should be enabled in addition to default plugins.
 	// These are called after default plugins and in the same order specified here.
+	// +listType=map
```

Comment:
Since order matters, you might want to leave these as atomic. Up to you?

Reply:
/hold

:thinking: so the validation libraries could potentially alter the order?

Reply:
Docs say order is preserved

https://github.com/kubernetes/kube-openapi/blob/master/pkg/idl/doc.go#L40-L51

Reply:
no-- but server side apply would let each item be owned separately. We defined what happens to the order in this case but it might be easier to understand if you make it atomic.

Reply:
Sounds good.
I'm setting atomic just for `Enabled` then.

Reply:
/hold cancel